.github/ISSUE_TEMPLATE/---enhancement-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F389 Enhancement request"
+about: Ask for something to be improved in this repo
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Please make sure that your request is in-scope for this repo.
+
+This project has a strictly limited scope: compiling Scala to JS, and interoperate with the ECMAScript language. Anything else is out of scope.
+
+Typical things that are out of scope include:
+
+* Can it be implemented as a library or as an external tool? If yes, it is out of scope.
+* Is it a missing Java library? If yes, check whether there is a separate library that provides it (e.g., `java.time`).
+* Is it a missing JavaScript library? Unless it is specified in ECMAScript itself, it is out of scope.
+* Is it a language feature request? Unless it addresses something of ECMAScript that cannot be done in Scala.js yet, it is out of scope.
```

Comment:
Hum, this might be a bit strongly worded: I think it is OK to accept new features that make things easier, if the overhead of doing so is much smaller in the core repo... Not sure how to improve the wording though if at all...

One example that I think fits this description but I think is in scope is support to build progressive apps.